### PR TITLE
Merge : Fixes to Mv2, tested and working,

### PR DESCRIPTION
**Pull request by punixcorn**
changes to error messages, fixes to bugs and random prompts
fully tested

### CODE REVIEW

**Comment by punixcorn** [comment]
forgot some folders..

**Comment by punixcorn** [comment]
ok fixed

**Review by SpeedyQweku — APPROVED** [review]
You are making me confuse

File: .src/mv2.c [hunk 1]
```diff
@@ -14,66 +14,91 @@
...
     strcat(command, " ");
-    strcat(command, *(argv + i));
+    if (*(argv + i) != NULL)
+      strcat(command, *(argv + i));
     i++;
   }
-
-#if dcommand
-  printf("Command : %s\nCheck is %d\n", command, check);
-  exit(0);
```
Comment (punixcorn) [inline .src/mv2.c:72]: forgot to remove this , but no issue

**Comment by punixcorn** [comment]
> You are making me confuse

what do you mean? i closed, fixed and reopened

**Pull request merged**